admin/pages/social.php

```diff
@@ -10,120 +10,53 @@
+		'screencast_video_url' => 'https://yoa.st/screencast-social-pinterest',
+	),
+	'google'     => array(
+		'label'                => __( 'Google+', 'wordpress-seo' ),
+		'screencast_video_url' => 'https://yoa.st/screencast-social-google',
+	),
+);
 
-		} ?>
 
-		<h4><?php esc_html_e( 'Default settings', 'wordpress-seo' ); ?></h4>
-		<?php $yform->media_input( 'og_default_image', __( 'Image URL', 'wordpress-seo' ) ); ?>
-		<p class="desc label">
-			<?php esc_html_e( 'This image is used if the post/page being shared does not contain any images.', 'wordpress-seo' ); ?>
-		</p>
 
-		<?php $social_facebook->show_form(); ?>
-
-		<?php do_action( 'wpseo_admin_opengraph_section' ); ?>
-	</div>
-
-	<div id="twitterbox" class="wpseotab">
-		<?php $yform->light_switch( 'twitter', __( 'Add Twitter card meta data', 'wordpress-seo' ) ); ?>
-		<p>
-			<?php
-			/* translators: %s expands to <code>&lt;head&gt;</code> */
-			printf( __( 'Add Twitter card meta data to your site\'s %s section.', 'wordpress-seo' ), '<code>&lt;head&gt;</code>' );
-			?>
-		</p>
-		<br/>
-		<?php
-		$yform->select( 'twitter_card_type', __( 'The default card type to use', 'wordpress-seo' ), WPSEO_Option_Social::$twitter_card_types );
-		do_action( 'wpseo_admin_twitter_section' );
-		?>
-	</div>
-
-	<div id="pinterest" class="wpseotab">
-		<p>
-			<?php _e( 'Pinterest uses Open Graph metadata just like Facebook, so be sure to keep the Open Graph checkbox on the Facebook tab checked if you want to optimize your site for Pinterest.', 'wordpress-seo' ); ?>
-		</p>
-		<p>
-			<?php _e( 'If you have already confirmed your website with Pinterest, you can skip the step below.', 'wordpress-seo' ); ?>
-		</p>
-		<p>
-			<?php
-				/* translators: %1$s / %2$s expands to a link to pinterest.com's help page. */
-				printf( __( 'To %1$sconfirm your site with Pinterest%2$s, add the meta tag here:', 'wordpress-seo' ), '<a target="_blank" href="https://help.pinterest.com/en/articles/confirm-your-website#meta_tag">', '</a>' );
-			?>
-		</p>
-
-		<?php $yform->textinput( 'pinterestverify', __( 'Pinterest confirmation', 'wordpress-seo' ) ); ?>
+?>
+	<h2 class="nav-tab-wrapper" id="wpseo-tabs">
+		<?php foreach ( $tabs as $identifier => $tab ) : ?>
+		<a class="nav-tab" id="<?php echo $identifier; ?>-tab" href="#top#<?php echo $identifier; ?>"><?php echo $tab['label']; ?></a>
+		<?php endforeach; ?>
+	</h2>
+<?php
 
-		<?php
-		do_action( 'wpseo_admin_pinterest_section' );
-		?>
-	</div>
+foreach ( $tabs as $identifier => $tab ) {
 
-	<div id="google" class="wpseotab">
-		<?php $yform->light_switch( 'googleplus', __( 'Add Google+ specific post meta data', 'wordpress-seo' ) ); ?>
+	printf( '<div id="%s" class="wpseotab">', $identifier );
 
-		<p><?php _e( 'If you have a Google+ page for your business, add that URL here and link it on your Google+ page\'s about page.', 'wordpress-seo' ); ?></p>
+	if ( ! empty( $tab['screencast_video_url'] ) ) {
```

Comment:
It seems like we are using this pattern multiple times, can/should we abstract this?